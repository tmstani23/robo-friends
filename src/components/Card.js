import React, {Component} from 'react';


class Card extends Component {
    render() {
        return (
            <div className ='bg-animate hover-bg-light-blue tc bw2 shadow5 bg-light-green dib br3 pa3 ma2 grow'>
                <img alt='robots' src={`https://robohash.org/${this.props.id}?200x200`}/>
                <h2>{this.props.name}</h2>
                <p>{this.props.email}</p>
                
            </div>
            
        )
    }
}

export default Card;