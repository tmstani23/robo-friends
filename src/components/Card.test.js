import {shallow} from 'enzyme';
import React from 'react';
import Card from './Card';
import renderer from 'react-test-renderer';

it('expect card component to render', () => {
    const tree = renderer.create(
        <Card />
      ).toJSON();
      expect(tree).toMatchSnapshot();
})
