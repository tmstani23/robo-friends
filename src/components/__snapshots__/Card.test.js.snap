// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect card component to render 1`] = `
<div
  className="bg-animate hover-bg-light-blue tc bw2 shadow5 bg-light-green dib br3 pa3 ma2 grow"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <h2 />
  <p />
</div>
`;
